hybrid/README.md

```diff
@@ -0,0 +1,72 @@
+# Salesforce Mobile SDK for Hybrid Package
```

Comment:
@rwhitley this is the README.md for the new forcehybrid - let me know if you see anything that needs to change.